pkg/common/utils/utils.go

```diff
@@ -48,8 +51,8 @@ func QueryVolumeUtil(ctx context.Context, m cnsvolume.Manager, queryFilter cnsty
 	var queryResult *cnstypes.CnsQueryResult
 	var err error
 	if useQueryVolumeAsync {
-		// AsyncQueryVolume feature switch is disabled.
-		queryResult, err = m.QueryVolumeAsync(ctx, queryFilter, querySelection)
+		// AsyncQueryVolume feature switch is enabled.
+		queryResult, err = m.QueryVolumeAsync(ctx, queryFilter, nil)
```

Comment:
Why is querySelection removed here? And if it returns more than 1000 records based on querySelection, don't we need pagination here?

Reply:
Right! The callers can pass nil or querySelection struct. Addressed this comment. Thanks

Reply:
But do we need pagination in `QueryVolumeUtil` as well, like how you introduced it for `QueryAllVolumeUtil`?

Reply:
No. Because, we want `QueryAsync` to return results similar to `QueryVolume` API and expect callers to implement their own pagination(for ex: fullSyncGetQueryResults in `pkg/syncer/util.go` ).

Added pagination for `QueryAllVolumeUtil` because we want `QueryAsync` to return results similar to `QueryAll` API. So, `QueryAsync` had to be invoked multiple times to return all the results

Hope this clarifies :)

Reply:
Okay, makes sense.